packages/core/src/server/helper.ts

```diff
@@ -47,6 +49,29 @@ const formatPrefix = (prefix: string | undefined) => {
   return `${hasLeadingSlash ? '' : '/'}${prefix}${hasTailSlash ? '' : '/'}`;
 };
 
+export const getRoutes = (options: { context: InternalContext }): Routes => {
```

Comment:
destruct `context` from options to make the code shorter.

```suggestion
export const getRoutes = ({ context } : { context: InternalContext }): Routes => {
```